sdks/aperture-js/package.json

```diff
@@ -10,7 +10,8 @@
     "grpc-client": "npm run build && node ./lib/client.js",
```

Comment:
Could you please bump the version of the SDK so that the changes can be released once merged?

Reply:
Addressed in [de8302d](https://github.com/fluxninja/aperture/pull/2961/commits/de8302d8597d54369f855c1f30fd780b408483ba)